### ISSUE TITLE
Icon for Visual Studio Code is changing

### ISSUE THREAD

**Issue by colemickens** [numixproject/numix-core#3902]
| Application name | Icon name | Desktop file name | Original icon |
| --- | :-- | :-- | :-- |
| `Visual Studio Code`| `code` | `code` | [see below] |

It's the icon for Visual Studio Code. It's in the process of changing. It has been updated for the "nightly" / "Insiders" builds.

More information:   https://github.com/Microsoft/vscode/issues/6607#issuecomment-324456824

**Comment by ghost** [comment]
at the moment numix is using the black background, bezel & tilted version - is it possible to use the flat  version instead? personally i think it looks very poor next to the other numix icons that go for a flat design

**Comment by palob** [comment]
That doesn't sound like our icon, can you please show a screenshot?

**Comment by ChrisEbner** [comment]
![screenshot from 2017-09-07 13-12-57](https://user-images.githubusercontent.com/6430561/30160810-64cc7a0e-93ce-11e7-9ac1-cd5dc6d7d324.png)

**Comment by Foggalong** [comment]
Yeah that's not a Numix icon, so we'll get an update for it out as soon as possible.

As a note though we don't theme to the icon name `code.svg` and instead link that to [our text editor icon](https://github.com/numixproject/numix-core/blob/master/data.json#L13824). We do this because the icon name is too generic to give something specific to vscode. Instead we have [these icons](https://github.com/numixproject/numix-core/blob/master/data.json#L14630-L14636) which are specific to vscode.

**Comment by ghost** [comment]
oh, i forgot to mention, this is related to code-insiders, not code. is it possible to get an updated icon for that aswell?

**Comment by Foggalong** [comment]
Oh right, sure! :smile: We'll need the icon line from the launcher for that one also

**Comment by ghost** [comment]
i'm not sure about your last response, do you need something from me in order to proceed? the icon has now changed, but is still the wrong tilted bezel version instead of the flat one:

![new icon](https://user-images.githubusercontent.com/11061334/31069272-0b5600e8-a75c-11e7-88e8-18815e903923.png)

**Comment by palob** [comment]
That's not a Numix icon at all which means your installation uses another icon name. You need to look up the icon name in the respective ``.desktop`` launcher file.

**Comment by ghost** [comment]
Okay, i think i've found the issue here: 
The ```Visual Studio Code.desktop``` and ```code-insiders.desktop``` files have ```code``` as their icon. 
According to #3504 its numix decission to not change the code icon, but i think the ```code-insiders```-icon from the ```code-insiders.desktop``` could be created? Currently there is no numix icon available for that, thats why its using the ```/usr/share/pixmaps/code-insiders.png``` instead - which is the tilted & bezeled one.

**Comment by palob** [comment]
Does ``code-insiders`` use the same icon?

**Comment by colemickens** [comment]
Code uses the orange variant of the icon.

Code-Insiders uses a green variant of the icon.

**Comment by jose-a-sa** [comment]
New information has arrived regarding the VS Code icon. They decided to use the new design with blu variant for stable and green for insiders https://code.visualstudio.com/blogs/2017/10/24/theicon

**Comment by bilelmoussaoui** [comment]
The icon has been changed for like 10 times :rofl: They kept the blue one in their latest version. I don't think it should be updated anyway; maybe just having a different colours for the insider version?

**Comment by Foggalong** [comment]
Yeah, tbh I think that was the only real take away from this. No enhancement for the [blue version](https://github.com/numixproject/numix-core/blob/master/icons/circle/48/visualstudiocode.svg), but an equivalent green version for `visualstuidiocode-insiders.svg`
